content/en/docs/instrumentation/java/manual.md

```diff
@@ -24,16 +24,32 @@ interface.
 If you are an application developer, you need to configure an instance of the
 `OpenTelemetrySdk` as early as possible in your application. This can be done
 using the `OpenTelemetrySdk.builder()` method.
+The returned `OpenTelemetrySdkBuilder` instance gets the providers related to the 
+signals, tracing and metrics, in order to build the `OpenTelemetry` instance.
+
+You can build the providers by using the `SdkTracerProvider.builder()` and
+`SdkMeterProvider.builder()` methods.
+It is also necessary to define a `Resource` instance as a representation of the
+entity producing the telemetry. It can bring attributes and the `service.name` is
+a mandatory one.
```

Comment:
It's not really "necessary"/"mandatory", you can create an SDK instance without it and you won't get any error. It's still highly recommended though:
```suggestion
It is also strongly recommended to define a `Resource` instance as a representation of the
entity producing the telemetry; in particular the `service.name` attribute is
the most important piece of telemetry source-identifying info.
```

Reply:
Yeah, it's a bad usage of "mandatory" ... I was thinking that without that attribute you don't have a way to filter in the Jaeger UI AFAIK.